<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrivClear</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_browser">
       <attribute name="title">
        <string>浏览器清理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget_browser_info">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_cookies">
            <property name="text">
             <string>选择Cookies</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_history">
            <property name="text">
             <string>选择历史记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_scan_browser">
            <property name="text">
             <string>扫描浏览器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_clean_browser">
            <property name="text">
             <string>清理浏览器</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_system">
       <attribute name="title">
        <string>系统清理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextBrowser" name="textBrowser_system_info"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_temp">
            <property name="text">
             <string>选择临时文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_logs">
            <property name="text">
             <string>选择LOG文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_scan_system">
            <property name="text">
             <string>扫描系统文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_clean_system">
            <property name="text">
             <string>清理系统文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_network">
       <attribute name="title">
        <string>网络清理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowser_network_info"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox_unuse">
            <property name="text">
             <string>选择未使用的网络</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_scan_network">
            <property name="text">
             <string>扫描网络配置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_clean_network">
            <property name="text">
             <string>清理网络配置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_reports">
       <attribute name="title">
        <string>日志</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutReports">
        <item>
         <widget class="QTextBrowser" name="textBrowser_report"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_generate_report">
          <property name="text">
           <string>生成清理日志</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
